src/constants/app-constants.ts

```diff
@@ -339,3 +347,16 @@ export const SALT_FOR_PASSWORD = '';
 export const MIN_PASSWORD_LENGTH = 8;
 
 export const COMPACT_SUMMARY_MAX_LENGTH = 200;
+
+export const DUMMY_HIDDEN_VALUE = '********';
+
+export const AUTH_TYPE_CONFIG = {
+    BASIC_AUTH : {
```

Comment:
We do not have keys name like this, it should be instead:
```suggestion
    basicAuth : {
```